Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -57,6 +57,17 @@ if(!$noDiscoveryService){
 }
 $identityServiceUrl = Get-ApplicationEndpoint -appName $installSettings.appName -applicationEndpoint $installSettings.applicationEndPoint -installConfigPath $installConfigPath -scope $installSettingsScope -quiet $quiet
 
+$idpssConfig = $installSettings.identityProviderSearchServiceConfig
```

Comment:
@ctediz - Do these changes conflict with some of yours?

Reply:
Yah, I remember unchecking it in the merge, yet here it is again duplicated. Thanks